plugins/generator-1.18.1/forge-1.18.1/templates/json/fluid_states.json.ftl

```diff
@@ -0,0 +1,7 @@
+{

```

Comment:
This should be more similar to what water and lava do: you should have a json file for the block model as well, but only include the "particles" texture, with no parent model

Reply:
The still texture should be good

Reply:
:heavy_check_mark: Fixed